app/scripts/styles/_uswds-theme.scss

```diff
@@ -14,9 +14,11 @@
     ),
   $theme-type-scale-md: 8,
   $theme-utility-breakpoints: (
-    'mobile-lg': false,
-    'desktop': false
+
+    'mobile-lg': true,
+    'desktop': true
   ),
   $theme-font-type-serif: baseFontFamily,
   $theme-font-type-sans: baseFontFamily
 );
```

Comment:
One item I want to call out @AliceR for the footer class to work out of the box I needed to turn on the breakpoints in the theme utility.

Reply:
I wonder why we had them set to false, and what the consequences are with the change?